playground/keycloak/Keycloak.AppHost/aspire-manifest.json

```diff
@@ -3,18 +3,11 @@
   "resources": {
     "keycloak": {
       "type": "container.v0",
-      "image": "quay.io/keycloak/keycloak:26.1",
+      "image": "quay.io/keycloak/keycloak:26.2",
       "args": [
         "start",
         "--import-realm"
       ],
-      "bindMounts": [
-        {
-          "source": "../realms",
-          "target": "/opt/keycloak/data/import",
-          "readOnly": false
-        }
```

Comment:
@danegsta this all of our bind mounts have turned into container files. That "breaks" deployment and we need to figure out the solution .